jaws-journey-deploy/README.md

```diff
@@ -155,9 +163,21 @@ Does not require any parameters passed through.  However it does require that yo
 This step generates a code coverage report and adds to your GitHub PR as comments.  To generate the report, the step [build-and-test](#build-and-test) need to of already completed.
  
 **Parameters**
+* send_slack_notification_on_fail: flag to indicate whether to send a slack notification when build fails (default true)
 
 Does not require any parameters to be passed.  It does however need the following environment variables to be set `GITHUB_BOT_USERNAME` and `GITHUB_BOT_PACKAGE_MANAGER_TOKEN`
 
+### Slack Notification
+**Description**
+This step will send a slack notification when a build step fails.  This is to increase visibility of failures to the development team.
+
+To run the slack notification we implemented the steps in the [developers setup guide](https://github.com/CircleCI-Public/slack-orb/wiki/Setup#4-create-a-context-on-circleci).  For CircleCi you just need to make sure you are passing the following environment variables
+- SLACK_ACCESS_TOKEN
+- SLACK_DEFAULT_CHANNEL - to get the channel Id it is easier to open slack in a web browser as you can see the channel Id in the url  
+
+#### Example notification
+![Notification Sample](./images/notification_sample.PNG)
+
 ## Example Usage
```

Comment:
Maybe add the new parameter to the example

Reply:
removed parameter